.github/actions/detect-solidity-file-changes/action.yml

```diff
@@ -19,18 +19,3 @@ runs:
             - '.github/workflows/solidity.yml'
             - '.github/workflows/solidity-foundry.yml'
             - '.github/workflows/solidity-wrappers.yml'
-          read_only_sol:
```

Comment:
Are these intended removal?

Reply:
Moved to `.github/actions/detect-solidity-readonly-file-changes/action.yml`